docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -59,8 +59,66 @@ alluxio.master.mount.table.root.ufs=<STORAGE_URI>
 - The second property `alluxio.master.mount.table.root.ufs` sets to the URI of the under store to
   mount to the Alluxio root.
   This shared storage system must be accessible by the master node and all worker nodes.
-  Examples include `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`, or 
-  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`.
+  
+  For example, when [HDFS]({{ '/en/ufs/HDFS.html#basic-setup' | relativize_url }})
+  is used as the under storage system, the value of this property can be set to
+  `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`
+  
+  When [Amazon S3]({{ '/en/ufs/S3.html#basic-setup' | relativize_url }})
+  is used as the under storage system, the value can be set to
+  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`
+
+Append thehost name of each node into conf/masters and conf/workers accordingly.
```

Comment:
`Append the hostname of each node`
format the file names `conf/masters` and `conf/workers` with backticks